### DISCUSSION TITLE
replace one file with another

### DISCUSSION THREAD

**Discussion by Alister-Zeus** [hsoft/dupeguru#1040] [Ideas]
say you have two files and one is a higher quality / size than the other. You should be able to replace the lower quality one with the higher one in one step

at present the process I have is

1. ensure no other files are marked
2. rename the higher quality one to the same name as lower one except using .jpeg instead of .jpg
3. move the higher quality one into the directory of lower quality one
4. as the pair disappears from the results once complete rerun the scan ensuring the directory moved into is on normal not reference

**Comment by Feejus** [comment]
This is great idea as well as same for video, if the meta data is read and it can be established that this movie or Jpg is the same or if the other picture or movie has greater quality then replace the old one.
